composer.json

```diff
@@ -23,7 +23,7 @@
         "sonata-project/core-bundle": "^3.4",
         "symfony/form": "^2.8 || ^3.2 || ^4.0",
         "symfony/http-kernel": "^2.8 || ^3.2 || ^4.0",
-        "twig/twig": "^1.34 || ^2.0"
```

Comment:
Why did you remove twig? We still need twig for our templates

Reply:
@core23 https://packagist.org/packages/symfony/twig-bundle requires twig

Reply:
That's right, but we can require other version of twig. Maybe a higher version than symfony does

Reply:
@core23 yes, some builds fail, if there is lowest version of twig 👎